src/components/multi-action-button/multi-action-button.style.ts

```diff
@@ -14,6 +16,17 @@ const StyledMultiActionButton = styled.div<StyledMultiActionButtonProps>`
   display: inline-block;
   position: relative;
 
+  ${({ width }) =>
+    width &&
+    css`
+    ${width && computeWidth({ width })}
```

Comment:
```suggestion
    ${computeWidth({ width })}
```

you don't need to check `width` as it's already guarded outside this (line 20)